tests/foreman/ui/test_host.py

```diff
@@ -591,19 +591,13 @@ def test_positive_create_with_puppet_class(session, module_host_template, module
     :CaseLevel: System
     """
     pc_name = 'generic_1'
-    cv = publish_puppet_module(
-        [{'author': 'robottelo', 'name': pc_name}],
-        CUSTOM_PUPPET_REPO,
-        organization_id=module_org.id,
-    )
-    env = (
-        entities.Environment()
-        .search(query={'search': f'content_view="{cv.name}" and organization_id={module_org.id}'})[
-            0
-        ]
-        .read()
-    )
-    env = entities.Environment(id=env.id, location=[module_loc]).update(['location'])
+    env_name = default_sat.create_custom_environment(repo=pc_name)
+    env = entities.Environment().search(query={'search': f'name={env_name}'})[0].read()
```

Comment:
You can use `default_sat.api.Environment()` here, slowly removing the need to import entities in the file.

Reply:
Please do

Reply:
I will, want to ask, why do I change it on one place here, if there is other 100 `entities.usage`. Should I do it all at the same time ?

Reply:
and I understand the concept, that from this day D we will slowly start to check all the new code to use new concept, but there still will be PR changing all other occurences.

I can see positive effect, if something do not work correctly with new changes introduced, it will impact only few places, and not whole automation. On the other hand will be those problem catched by us ?